io/plugins/eu.esdihumboldt.hale.io.xls/src/eu/esdihumboldt/hale/io/xls/AbstractAnalyseTable.java

```diff
@@ -140,13 +144,15 @@ protected void analyseHeader(Sheet sheet) {
 	 * 
 	 * @param sheet the table sheet
 	 */
-	private void analyseContent(Sheet sheet) {
+	private void analyseContent(Sheet sheet, int skipNlines) {
+		int startingLine = 0;
```

Comment:
My understanding was that if you have 0 as skip lines then the first line is the header. 
but it's very true that you might have already the first instance in the first line. 

In the case of having an header this one would be added twice: once to the header and once to the rows, so you would add 2 instances for the same header. 
but in case you have no header, you need to consider also the first line in the content.

Why we don't add a checkbox in case there is a header and make the life easier to the user to understand and also to us?

Reply:
If 0 lines should be skipped then the first line/row already contains data and no header.

For one thing we can't just change the behavior of this setting and thus create backwards compatibility issues.
If we feel there is the need to improve the user experience that should be part of a separate task.

Reply:
Done. now works with all the possible cases.
Thank you for your patience :-)